pandas/core/format.py

```diff
@@ -241,7 +242,10 @@ def to_string(self):
             fmt_values.insert(row_num + n_header_rows, dot_str)
             fmt_index.insert(row_num + 1, '')
 
-        result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        if self.index:
+            result = self.adj.adjoin(3, *[fmt_index[1:], fmt_values])
+        else:
```

Comment:
is this really what dataframe does ? (e.g. with the adjoin)? for the no index case